script/DeploymentConfig.s.sol

```diff
@@ -0,0 +1,48 @@
+//// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.17;
+
+import { Script } from "forge-std/Script.sol";
+
+contract DeploymentConfig is Script {
+    struct TokenConfig {
+        string name;
+        string symbol;
+        string baseURI;
+        bytes signerPublicKey;
+    }
+
+    TokenConfig public ownerTokenConfig;
+    TokenConfig public masterTokenConfig;
+
+    address public deployer;
+
+    constructor(address _broadcaster) {
+        if (block.chainid == 31_337) {
+            (ownerTokenConfig, masterTokenConfig) = getOrCreateAnvilEthConfig();
+        } else {
+            revert("no network config for this chain");
+        }
```

Comment:
At the moment, deployment of these contracts happens inside Status clients, so we only have a deployment config for a local test node here